structured_experiments/2022.04.28.notebook_template/experiment.MD

```diff
@@ -0,0 +1,41 @@
+<!--DESC Template folder to use for experiments-->
+# Hypothesis
+TODO: Create Template
+
+# Methodology
+
+TODO: Create Template
+
+# Results
+
+TODO: Create Template
+
+# Conclusions
+
+TODO: Create Template
+
+
+
+# Reproduce
+
+## 1. First Acquire the training data
+
+### 
+Run `git lfs pull --exclude .git -I  data/storysquad-ground-truth/`

```

Comment:
this needs to be changed to tell the user to run `git lfs pull --exclude .git -I data/storysquad-ground-truth/` and `git lfs pull --exclude .git -I data/kaggle-ground-truth/`